src/functions/postResult/application/examiner-check-service.ts

```diff
@@ -0,0 +1,15 @@
+import { StandardCarTestCATBSchema } from '@dvsa/mes-test-schema/categories/B';
+import jwtDecode = require('jwt-decode');
+
+export const examinerCheck = (testResult: StandardCarTestCATBSchema, jwt: string): boolean => {
+  if (!testResult || !jwt) {
+    return false;
+  }
+  const decodedJWT = JSON.stringify(jwtDecode(jwt));
+  const resultStaffNumber = testResult.journalData.examiner.staffNumber;
+  const searchQuery = `"extn.employeeId":["${resultStaffNumber}"]`;
```

Comment:
The format and key of `extn.employeeId` can vary between the different instances of DVSA AAD. You will need `extn.employeeId` to be configurable and also accept different formats, check this PR for reference:

https://github.com/dvsa/mes-journal-service/pull/26